apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/AplDbVersion.java

```diff
@@ -732,13 +732,9 @@ protected int update(int nextUpdate) {
             case 270:
                 apply("CREATE TABLE IF NOT EXISTS dex_offer (db_id IDENTITY AUTO_INCREMENT, transaction_id BIGINT NOT NULL, type TINYINT NOT NULL," +
                         "account_id BIGINT NOT NULL, offer_currency TINYINT NOT NULL, offer_amount BIGINT NOT NULL, pair_currency TINYINT NOT NULL," +
-                        " pair_rate DECIMAL NOT NULL, finish_time INT NOT NULL)");
+                        " pair_rate DECIMAL NOT NULL, finish_time INT NOT NULL, status TINYINT NOT NULL, height INT NOT NUL, latest BOOLEAN NOT NULL DEFAULT TRUE)");
```

Comment:
Database will be inconsistent with this sql updates.

Reply:
Yes, but this changes (dex_offer table) is not on the prod.